src/lua/api-gateway/util/OauthClient.lua

```diff
@@ -16,22 +16,53 @@ function OauthClient:new(o)
     return o
 end
 
+local dogstats = require "api-gateway.dogstatsd.Dogstatsd"
+local dogstatsInstance = dogstats:new()
+
+--- metrics for Dogstatsd
+local httpCalls = 'oauth.http_calls'
```

Comment:
rename it to oauthHttpCallsMetric

Reply:
i noticed that oauth.http_calls is used as a metric name and also as a namespace (helps creating other metric names). WDYT if instead of using a local variable, you would add a "field" in OauthClient class and in this way reusing it in other classes that inherit OauthClient?